models/models.js

```diff
@@ -23,11 +23,14 @@ exports.selectEndpoints = () => {
 }
 
 
-exports.selectArticles = (sort_by = 'created_at', order = 'DESC') => {
+exports.selectArticles = (sort_by, order, topic) => {
     const new_sort_by = sort_by || 'created_at'
     const new_order = order || 'DESC'
+    const new_topic = topic || 'everything'
+
     const validSortByColumns = ['created_at', 'votes', 'title', 'author', 'topic', 'article_id']; 
     const validOrderValues = ['asc', 'desc'];
+    const validTopicValues = ['mitch', 'cats', 'paper']
```

Comment:
Hard code here will pass the tests, but if you connect to a different DB or if the api allows posting new topics, there may be topics are not included in this list. Think about a way to make it dynamic